Source/Flow/Public/Types/FlowDataPinBlueprintLibrary.h

```diff
@@ -0,0 +1,185 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "FlowDataPinProperties.h"
+#include "FlowDataPinResults.h"
+
+#include "FlowDataPinBlueprintLibrary.generated.h"
+
+// Auto-cast operators for blueprint to their inner types
+UCLASS()
+class UFlowDataPinBlueprintLibrary : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
+public:
+
+	// Recommend implementing AutoConvert_FlowDataPinProperty... for every EFlowPinType
+	FLOW_ASSERT_ENUM_MAX(EFlowPinType, 12);
+
+	// FFlowDataPinProperty auto-cast functions
+
+	// Convert bool property values to their inner blue values
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Convert to Bool", CompactNodeTitle = "->", BlueprintAutocast), Category = DataPins)
```

Comment:
Hellloooo boilerplate.

These auto-cast functions provide an auto-cast mini-node in blueprint to get the value out of the wrapper struct and into the desired blueprint native type.